staging/src/k8s.io/sample-apiserver/artifacts/example/rbac.yaml

```diff
@@ -0,0 +1,11 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: aggregated-apiserver-clusterrole
```

Comment:
This is actually generic to all apiservers.  Let's make an aggregated clusterrole that combines this the auth delegator. @kubernetes/sig-auth-misc

Reply:
Created the issue #65442 for further discussion.